crypto-hash-example/flb.conf

```diff
@@ -0,0 +1,20 @@
+[SERVICE]
+    flush      0.1
+    grace      2
+    log_level  info
+
+[INPUT]
+    name tail
+    read_from_head true
+    exit_on_eof true
+    path data.log
+
+[FILTER]
+    Name    lua
+    Match   *
+    script  filter.lua
```

Comment:
We could also have this all as inline LUA in YAML with 1.9...

Reply:
Sure, we can do in a future iteration, too many changes in this PR already :D